agent/api/container/port_binding.go

```diff
@@ -30,9 +32,13 @@ const (
 // PortBinding represents a port binding for a container
 type PortBinding struct {
```

Comment:
for each port binding, we know that either one of `containerPort` or `containerPortRange` is set, not both.

hence below, `ContainerPort` is changed from `uint16` -> `*uint16` and `containerPortRange` is also a `*string` because we want to do a nil check on these while populating v4 task metadata response.